src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -76,7 +76,7 @@ static Library()
                 StandardErrorHandling<NumberValue>(
                     BuiltinFunctionsCore.Abs.Name,
                     expandArguments: NoArgExpansion,
-                    replaceBlankValues: ReplaceBlankWithZero,
+                    replaceBlankValues: NoOpAlreadyHandledByIR,
```

Comment:
>            replaceBlankValues: NoOpAlreadyHandledByIR, [](http://example.com/codeflow?start=8&length=55)

Note - the fact that we're removing these from Library and still passing the tests is good coverage.